File: Neomer.EveryPrice.SDK/Helpers/BlobHelper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Neomer.EveryPrice.SDK.Helpers
{
	public static class BlobHelper
	{
		public static String Encode(byte[] data)
		{
			return Convert.ToBase64String(data);
		}

		public static byte[] Decode(String encodedData)
		{
			return Convert.FromBase64String(encodedData);
		}
	}
}

File: Neomer.EveryPrice.Android/app/src/main/java/com/neomer/everyprice/core/helpers/BlobHelper.java
package com.neomer.everyprice.core.helpers;

import android.util.Base64;

public final class BlobHelper {

    public static String Encode(byte[] data) {
        return Base64.encodeToString(data, Base64.URL_SAFE);
    }

    public static byte[] Decode(String encodedData) {
        return Base64.decode(encodedData, Base64.URL_SAFE);
    }

}
